documentation/samplefiles/readme.go.md

```diff
@@ -12,35 +12,15 @@ go:
 
 ``` yaml $(go) && $(multiapi)
 batch:
-  - tag: package-2019-12-01
-  - tag: package-2020-07-01-preview
-  # add every tag listed below
+  - tag: package--[[Version]]-[[ReleaseState]]
 ```
 
-### Tag: package-2019-12-01 and go
+### Tag: package-[[Version]]-[[ReleaseState]] and go
 
-These settings apply only when `--tag=package-2019-12-01 --go` is specified on the command line.
-Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+These settings apply only when `--tag=package-[[Version]] --go` is specified on the command line.
+Please also specify `--go-sdks-folder=<path to the root directory of your azure-sdk-for-go clone>`.
 
-```yaml $(tag) == 'package-2019-12-01' && $(go)
-# NOTE: a stable tag must not contain any preview swagger files included
-# NOTE: go namespace can only consist of lower case letters, numbers and underscores
-namespace: yourservicename
-# NOTE: for special cases, you can hard code the namespace in the output-folder
-# NOTE: the output-folder of a stable tag must not be under the preview subdirectory
-output-folder: $(go-sdk-folder)/services/$(namespace)/mgmt/2019-12-01/$(namespace)
-```
-
-### Tag: package-2020-07-01-preview and go
-
-These settings apply only when `--tag=package-2020-07-01-preview --go` is specified on the command line.
-Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
-
-```yaml $(tag) == 'package-2020-07-01-preview' && $(go)
-# NOTE: a preview tag must not contain at least one preview swagger file included
-# NOTE: go namespace can only consist of lower case letters, numbers and underscores
-namespace: yourservicename
-# NOTE: a preview api-version must be under the preview sub-directory 
-# NOTE: the output-folder of a preview tag must be under the preview subdirectory
-output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2020-07-01-preview/$(namespace)
+```yaml $(tag) == 'package-[[Version]]-[[ReleaseState]]' && $(go)
+namespace: [[ServiceName]]
+output-folder: $(go-sdk-folder)/services/[[ReleaseState]]/$(namespace)/mgmt/[[Version]]-[[ReleaseState]]/$(namespace)
```

Comment:
Is there a way to make the output like `#1` when `ReleaseState` is `stable` and like `#2` when `ReleaseState` is `preview`?
1. `$(go-sdk-folder)/services/$(namespace)/mgmt/[[Version]]-[[ReleaseState]]/$(namespace)`
1. `$(go-sdk-folder)/services/preivew/$(namespace)/mgmt/[[Version]]-[[ReleaseState]]/$(namespace)`

Reply:
resolved and I did an example for this. You can see the preview version in https://github.com/JackTn/azure-rest-api-specs/blob/dev-attestation-Microsoft.Attestation-2020-09-21/specification/attestation/resource-manager/readme.go.md
and stable version in  https://github.com/JackTn/azure-rest-api-specs/blob/dev-tianenapimanagement-Microsoft.TianenApiManagement-2020-09-27/specification/tianenapimanagement/resource-manager/readme.go.md. pls check it.